ui/src/app/shared/header/chips/status-selection.component.ts

```diff
@@ -0,0 +1,32 @@
+import {JobStatus} from "../../model/JobStatus";
+import {Component, EventEmitter, Input, OnInit, Output} from "@angular/core";
+
+@Component({
+  selector: 'jm-status-selection',
+  templateUrl: './status-selection.component.html',
+})
+export class StatusSelectionComponent implements OnInit {
```

Comment:
When I enter a new "statuses" chip directly, looks like it's getting initialized with null or undefined; the API call then fails when I try to enter it.

I wasn't planning to do this during the demo anyways, but something to fix for this PR.

Reply:
Good catch, fixed